search/searcher/search_phrase.go

```diff
@@ -322,9 +338,15 @@ func findPhrasePaths(prevPos uint64, ap search.ArrayPositions, phraseTerms [][]s
 
 			// if enough slop remaining, continue recursively
 			if prevPos == 0 || (remainingSlop-dist) >= 0 {
+				for _, ppart := range p {
```

Comment:
maybe put a comment here explaining this identifies already used terms in the phrase?

Reply:
will do